src/hb-ot-meta-table.hh

```diff
@@ -42,7 +42,7 @@ struct DataMap
 {
   int cmp (hb_tag_t a) const { return tag.cmp (a); }
 
-  hb_tag_t get_tag () const { return tag; }
+  hb_ot_meta_t get_tag () const { return (hb_ot_meta_t) (hb_tag_t) tag; }
```

Comment:
The way I mentally think about this is, the `meta` table implementation only returns `hb_tag_t`, and `hb_ot_meta_t` is a detail of the public API.  So, the cast belongs to the API implementation to me.

Also, maybe better name for `hb_ot_meta_t`?  We have `hb_ot_name_id_t`.  `hb_ot_meta_tag_t` sounds better to me.

Reply:
Done